fastparse/shared/src/main/scala/fastparse/Api.scala

```diff
@@ -12,6 +12,8 @@ trait Api{
 
   val Result = core.Result
   type Result[+T] = core.Result[T]
+  val Failure = core.Result.Failure
+  val Success = core.Result.Success
```

Comment:
Ok, the name-clash with scala.util.[Try|Success|Failure] is to be avoided. So, `Result` -> `Parsed` is fine. Second, we follow the terminology of having `ed` names for results (e.g.`traced` in Extra).